src/lib/fragments/CheckboxList.react.js

```diff
@@ -51,7 +51,7 @@ const CheckboxList = (props)=>{
         >
             {sanitizeOptions(options).map((option, index) => {
                 return (
-                    <div className="govuk-checkboxes__item" key={option.value}>
+                    <div className="govuk-checkboxes__item" key={option.value} style={{paddingBottom:"10px"}}> 
```

Comment:
Would it be better to define a css class for this so that isn't hardcoded in the react component?

Reply:
good idea